README.md

```diff
@@ -41,7 +42,7 @@ available at [http://localhost:3000/](http://localhost:3000/).
 
 #### Unit Test
 
-The unit tests are powered by [chai](http://chaijs.com/) and [mocha](http://mochajs.org/).
+The unit tests are powered by [Jest](https://facebook.github.io/jest/), [Enzyme](http://airbnb.io/enzyme/) and [Sinon.JS](http://sinonjs.org/)
```

Comment:
I'm not super keen on the fact that we are introducing a new framework for the unit tests and this is different from the framework used for the e2e tests. Can we make use of the same frameworks?   From what I see in the code below there's no need to change the framework.